doc/changelog.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* :feature:`712` Add ``--pdb-filter`` command line argument for filter the cases which runs debugger
```

Comment:
`Added ``--pdb-filter`` - a new command-line flag that allows the user to enter pdb only on specific caught exceptions, based on pattern matching (similar to ``-k``)`

Reply:
Fixed